src/UglyToad.PdfPig.Fonts/TrueType/Parser/TableParser.cs

```diff
@@ -25,7 +25,7 @@ internal static class TableParser
 
             if (sum != table.CheckSum)
             {
-                Trace.TraceWarning("Table with invalid checksum found in TrueType font file.");
+                Debug.Write("Table with invalid checksum found in TrueType font file.");
```

Comment:
Cool, didn't know you could just do `Write`, I thought it was asserts only, thanks

Reply:
I meant to use `WriteLine` and not `Write`. I've changed that now